modules/sandboxed-containers-create-kata-config-resource-cli.adoc

```diff
@@ -0,0 +1,79 @@
+//Module included in the following assemblies:
+//
+// * sandboxed_containers/deploying_sandboxed_containers.adoc
+
+[id="sandboxed-containers-create-kataconfig-rsource-cli_{context}"]
+= Creating the KataConfig custom resource using the CLI
+
+You must create one `KataConfig` custom resource (CR) to install the Kata runtime on your nodes. When you create the `KataConfig` CR, it triggers the {sandboxed-containers-operator} to do the following:
```

Comment:
I'd avoid using "it". Maybe say "Creating the `KataConfig` CR triggers the..."

Reply:
Done - good comment :)